### ISSUE TITLE
Inventory Tool for collection and Distribution

### ISSUE THREAD

**Issue by astrogexo** [SoPR/maria-tech-brigade-effort#20]
Grassroots organizations in the US need a system to create an inventory of donations before shipping.  This will help speed up distribution once the shipment arrives in PR. Furthermore, NGOs on the ground will need to know real-time what is arriving and what they're distributing.

I propose a browser-based barcode scanner linked to a google spreadsheet for simplicity, familiarity, and ease of access.

The tool should be easy to use on a mobile device and allow for both image and manual input of inventory.

**Comment by astrogexo** [comment]
Found this
https://github.com/zxing/zxing.

**Comment by kalquaddoomi** [comment]
You might be better off with a React Native project for this, linked to an API for synchronization and a local database on the device, rather than a browser-based barcode scanner which would require a bit of work. I've worked with Zebra Crossing before and it really is specifically for Native Apps (it has implementations for Android and iOS).

**Comment by astrogexo** [comment]
Yes, ideally it would be native but I don't want to haven't people need to install an app.

Maybe that's not a big barrier but from what I've seen on the ground a lot of people running the collection drives aren't technically savvy

⁣Sent from BlueMail ​
